src/main/kotlin/apply/domain/recruitment/Recruitment.kt

```diff
@@ -11,6 +11,9 @@ class Recruitment(
     @Column(nullable = false)
     var title: String,
 
+    @Column(updatable = false)
+    val term: Long? = null,
```

Comment:
```suggestion
    term: Long? = null,
) : BaseEntity(id) {

    @Column(updatable = false)
    val term: Long = term ?: 1
```

`1회차 모집의 term만 null`이라는 걸 이용하면 이렇게도 나타낼 수 있을 거 같아요. 도메인 바깥에서 봤을 때 회차가 null이 아니면 좋을 것 같아서 한 번 던져봅니다.

Reply:
좋은 팁 감사합니다.
의견을 참고하겠습니다!! :)